<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTaskWindow</class>
 <widget class="QWidget" name="NewTaskWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <widget class="QPushButton" name="ConfirmAddTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ProjectTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Task</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TaskTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TaskDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Description</string>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ReccuringTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reccuring</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Deadline">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Deadline</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
